contracts/deploy/092_vault_admin_upgrade.js

```diff
@@ -0,0 +1,40 @@
+const {
+  deploymentWithGovernanceProposal,
+  deployWithConfirmation,
+} = require("../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "092_vault_admin_upgrade",
+    // forceDeploy: false,
+    // forceSkip: true,
+    // onlyOnFork: true, // this is only executed in forked environment
+    reduceQueueTime: true, // just to solve the issue of later active proposals failing
+    proposalId: "",
+  },
+  async ({ ethers }) => {
+    const cOETHVaultProxy = await ethers.getContract("OETHVaultProxy");
+    const cVaultAdmin = await ethers.getContractAt(
+      "IVault",
+      cOETHVaultProxy.address
+    );
+
+    // Deploy VaultAdmin implementation
+    await deployWithConfirmation("OETHVaultAdmin");
+    const dVaultAdminImpl = await ethers.getContract("OETHVaultAdmin");
+
+    return {
+      name: "Upgrade OETH Vault\n\
+    \n\
+    Part of simplified OETH proposal. This adds the ability to remove a supported asset from the Vault. \
+    ",
+      actions: [
+        {
+          contract: cVaultAdmin,
+          signature: "setAdminImpl(address)",
+          args: [dVaultAdminImpl.address],
+        },
+      ],
+    };
```

Comment:
can we add the removal of frxETH to this deployment?

Reply:
We can do it, but we need strategist to do swap the frxETH it holds. Right now, this PR allows dust upto `e12`. But the Vault right now has a little over `1e16` frxETH

Reply:
I've added the remove of frxETH to the deploy script as the remaining frxETH in the Vault has been swapped out for WETH